client/webserver/site/src/html/register.tmpl

```diff
@@ -71,6 +77,9 @@
           Enter your app password to confirm DEX registration.
           When you submit this form, <span id="feeDisplay"></span> DCR will be spent from your Decred wallet to pay registration fees.
         </div>
+        <div id="restorationMsg" class="d-hide">
+          If the DEX server indicates during registration that it already knows our HD account number, no fee will be necessary.
+        </div>
```

Comment:
I'm not crazy about this UX. We should really just check whether the server knows our seed-derived key as soon as we know their pubkey after `/getdexinfo`, and skip this form altogether. Unfortunately, I didn't see a way to do that without opening a can of worms.

Reply:
I opened the can. Might want to hold off on reviews for a minute.